### PR TITLE
Responsive hero

### PR DESCRIPTION
**Pull request by sskylar**
This PR changes so the giant hero is only used on 2xl screens, otherwise uses the mobile version (which has generous top bleed).

Comparison:
https://joinmastodon-git-responsive-hero55-oakstudios.vercel.app
vs
https://joinmastodon.vercel.app

### CODE REVIEW

File: components/Hero.tsx [hunk 1]
```diff
@@ -37,31 +37,32 @@ const Hero = ({
...
         <div className="full-width-bg__inner grid py-20 lg:grid-cols-12 lg:justify-center lg:gap-x-gutter">
-          <div className="xxl:col-span-5 col-span-12 lg:col-span-7 xl:col-span-5 xl:col-start-2">
+          <div className="col-span-12 lg:col-span-7 xl:col-span-5 xl:col-start-2">
             {children}
           </div>
         </div>
       )}
 
-      <div className="absolute inset-0 -z-10 xl:hidden">
+      <div className="absolute inset-0 -z-10 2xl:hidden md:ml-[-25%]">
```
[the remark this answers is not in the document]
Comment (danielgamage) [reply components/Hero.tsx:46]: mm we could scope that to the homepage (or maybe for more nuanced art direction we can pass in bespoke classNames (like `object-position`s) to the image at different sizes when we call `<Hero>`?)

**Review by danielgamage — COMMENTED** [review]
do we want this system for the inner pages as well, or no? we need some pretty different illustration assets if so.

one other thing: we calculate the height of the hero based on the height of the image, but we're going to need illustrations to be 1536px*2 (?) wide for them to work up to `2xl`, so the `--__-hero-height` variables may need to be calculated differently? (they might be fine too, just to keep in mind)

**Comment by mikabusante** [comment]
i personally like the 1st version (on https://joinmastodon-git-responsive-hero55-oakstudios.vercel.app/), since we can still get the original illo framing on larger screen sizes (and i think it looks nice :)!)

if we are scoping this to the homepage, we already pass the `large` prop, so we can switch when the desktop breakpoint applies like 
```
 className={classNames(
        "full-width-bg relative h-[var(--mobile-hero-height)] pt-[var(--header-area)] text-white",
        large
          ? "2xl:h-[var(--desktop-hero-height)]"
          : "xl:h-[var(--desktop-hero-height)"
      )}
```

and something similar on the images for when they are hidden/shown (as well as the `md:ml-[-25%]`). Tho it might make more sense to just rename the `large` prop to `home` since all these illo differences + the text layout are really specific to that page.

**Comment by danielgamage** [comment]
i'll push the scoping changes to this branch and merge it

**Pull request merged**